whelktool/scripts/2018/10/remodel-termComponentList.groovy

```diff
@@ -19,7 +19,13 @@ boolean changeToSubdivision(it, subdivisionTypes) {
     return changed
 }
 
-process { data ->
+selectBySqlWhere('''
+        data::text LIKE '%"termComponentList"%'
+        ''') { data ->
+    // guard against missing entity
+    if (data.graph.size() < 2) {
```

Comment:
Just for my own understanding, when can this happen?

Reply:
AFAIK, only when the LIKE select finds the vocab itself (which describes "termComponentList"), which is its own mainEntity (sort of). I opted for a guard over tuning the select to be more robust.